os/generic_win_base.h

```diff
@@ -25,14 +25,6 @@
 typedef int mode_t;
 typedef void pthread_attr_t;
 
-#if defined(__cplusplus)
```

Comment:
Should this instead just be added to `os/generic_unix_base.h` instead? This seems a platform specific thing instead of moving it to os/object.h - this declaration has nothing to do with os objects.

Reply:
I moved it to a new `<os/generic_base.h>` header.